idea/idea-fir/tests/org/jetbrains/kotlin/idea/quickfix/AbstractHighLevelQuickFixTest.kt

```diff
@@ -7,20 +7,90 @@ package org.jetbrains.kotlin.idea.quickfix
 
 import com.intellij.codeInsight.intention.IntentionAction
 import com.intellij.codeInspection.InspectionProfileEntry
+import com.intellij.openapi.util.io.FileUtil
 import org.jetbrains.kotlin.test.uitls.IgnoreTests
+import java.io.File
 import java.nio.file.Paths
 
 
 abstract class AbstractHighLevelQuickFixTest : AbstractQuickFixTest() {
+    companion object {
+        private val isTeamCityBuild: Boolean = System.getProperty("TEAMCITY_VERSION") != null
+    }
+
     override fun doTest(beforeFileName: String) {
         IgnoreTests.runTestIfEnabledByFileDirective(
             Paths.get(beforeFileName),
-            enableTestDirective = IgnoreTests.DIRECTIVES.FIR_COMPARISON_MUTLTILINE_COMMENT,
+            enableTestDirective = IgnoreTests.DIRECTIVES.FIR_COMPARISON_MULTILINE_COMMENT,
             directivePosition = IgnoreTests.DirectivePosition.LAST_LINE_IN_FILE,
             additionalFilesExtensions = arrayOf("after")
         ) {
             super.doTest(beforeFileName)
+            // getFir/OriginalAfterFile expect just the filename (no directory), but beforeFileName has the directory
```

Comment:
Nit: to be clearer, how about `// get(Fir|Original)AfterFile expect...` ?

Reply:
I just spelled out both.